lib/fastlane/plugin/firebase_app_distribution/helper/upload_status_response.rb

```diff
@@ -3,35 +3,54 @@ def initialize(response_json_hash)
     @response_json_hash = response_json_hash
   end
 
+  def done
+    !!@response_json_hash[:done]
+  end
+
+  def response
+    @response_json_hash[:response]
+  end
+
+  def release
+    response ? response[:release] : nil
+  end
+
+  def release_id
```

Comment:
this method should not be needed once working with resource names

Reply:
Done.